src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -135,6 +135,26 @@ class KotlinModule @Deprecated(
         addMixIn(CharRange::class.java, ClosedRangeMixin::class.java)
         addMixIn(LongRange::class.java, ClosedRangeMixin::class.java)
         addMixIn(ClosedRange::class.java, ClosedRangeMixin::class.java)
+
+        addComponentsForKotlin_1_5(context)
+    }
+
+    @Suppress("FunctionName")
+    private fun addComponentsForKotlin_1_5(context: SetupContext) {
+        if (KotlinVersion.CURRENT.isAtLeast(1, 5)) {
+            try {
+                val kClass =
+                    Class.forName("com.fasterxml.jackson.module.kotlin.KotlinModuleVersionAdapter")
```

Comment:
I take it this can't be a class reference (which would survive refactoring)?

Reply:
No, because the class doesn't exist when compiling with Gradle using The 1.3 version of Kotlin. I could theoretically work around this by compiling with one version of Gradle but running the tests against a different version at runtime.

Reply:
Nah, I think that's fine